packages/cerebral-provider-forms/src/utils/validate.js

```diff
@@ -34,13 +34,16 @@ export default function validate (form, value, validationRules) {
       arg = undefined
     }
 
+    
+    const isValid = rule(value, form, arg)
+
     return {
```

Comment:
Hi @ArnoBuschmann, 
If field is valid, I think it is better to return initialValidation.
 `return isValid ? initialValidation : {
  .....
}`